src/include/reconcile.i

```diff
@@ -197,6 +197,24 @@ __rec_page_time_stats(WT_SESSION_IMPL *session, WT_RECONCILE *r)
 static inline bool
 __wt_rec_need_split(WT_RECONCILE *r, size_t len)
 {
+    uint32_t page_items;
```

Comment:
This is the relevant part of the diff.
The other stuff is because I've branched off `wt-6444-abort-old-txn` and pulled from `develop`.